src/content/web/get-started.md

```diff
@@ -188,7 +188,6 @@ Check out these resources:
   * [Introduction to the DOM][]
 * Tutorials and codelabs for Dart
   * [Tutorials](/tutorials)
```

Comment:
**issue:** Same as [line 2778](https://github.com/dart-lang/site-www/pull/5858/files#diff-75ed5ce4cf7ea66ddd2f5aa89546afe28498b1afb33bcf6a6f56fb7407445cd6R2778) in [js-to-dart.md](src/content/resources/coming-from/js-to-dart.md)

```suggestion
* Dart [tutorials](/tutorials)
```